src/PathUtils.cpp

```diff
@@ -57,9 +57,13 @@ fs::path PathUtils::GetCachePath()
 
 fs::path PathUtils::GetRoamingDataPath()
 {
+#if TARGET_OS_TV
+    return GetCachePath();
```

Comment:
nit: Can you use a tab to indent this line? Thanks!

Reply:
Yes, absolutely. I pushed an updated PR at the same time that the suggested change below came in. Thanks!

Reply:
Great! Thanks again!